holesky/.env.example

```diff
@@ -13,6 +13,9 @@ NODE_NUM_WORKERS=1
 NODE_DISPERSAL_PORT=32005
 NODE_V2_DISPERSAL_PORT=32006
 
+# This flag defines a runtime mode. Acceptable inputs are v1, v2, v1-and-v2
+NODE_RUNTIME_MODE=v1
```

Comment:
Valid inputs are `v1-only, v2-only, v1-and-v2`. The default runtime mode is `v1-and-v2`
```
> ./bin/node --help | grep RUNTIME_MODE
   --node.runtime-mode value                             Node runtime mode (v1-and-v2 (default), v1-only, or v2-only) (default: "v1-and-v2") [$NODE_RUNTIME_MODE]
```